store/postgres/src/relational.rs

```diff
@@ -767,6 +825,12 @@ mod tests {
             bigThing: Thing!
         }
 
+        enum Color {
+            yellow,
+            red,
+            BLUE
+        }
```

Comment:
Can we have a test with two different enum types? That would catch the HashMap ordering thing.

Reply:
Ok